### DISCUSSION TITLE
how to deploy sergei strelec on network boot?!

### DISCUSSION THREAD

**Discussion by smtadmin67** [ipxe/ipxe#980] [General]
hi dears
would you explain how to deploy sergei strelec in network boot?
i downloaded sergei and want use it in ipxe script for the IT Technician that boot workstations to use sergei strelec on pxe boot
do you have any ideas or exprience?!

**Comment by NiKiZe** [comment]
That is not an error in iPXE, which means it don't belong under issues but rather discussions, and moved accordingly.

I have no idea what sergei strelec is but a quick search indicates winpe.
There is wimboot, which can be used to boot .wim files.
But 4.5gb in wim would be quite hard on memory usage, and if not all in wim, you will get issues with accessin "out of wim" files.

This is really a question for the "distro community" that is the sergei strelec community in this case. Ask them how to netboot, but feel free to mention iPXE and wimboot, and maybe they have a solution for you.

**Comment by kcinickgx** [comment]
#!ipxe
sanboot --drive 0x80 http://${domain}/boot/sergei/WinPE11_10_Sergei_Strelec_x64_2025.05.22_English.iso

**Reply by ghost** [reply]
strelec11x64Eng.wim ??

**Comment by lucasddev00-svg** [comment]
I did deploy on Windows Server 2022, with a WDS, but every time i need to use all function from sergei, i got to plug a pendrive with sergei installed. So it wont work so good for me.

**Reply by lucasddev00-svg** [reply]
im using the .wim file from folder source.
